### PR TITLE
feat(amplify): add new subcommands

### PR DESCRIPTION
**Pull request by adilusman51**
add new sub commands to amplify cli

### CODE REVIEW

File: src/amplify.ts [hunk 1]
```diff
@@ -57,13 +57,18 @@ const categoryCommands: Fig.Subcommand[] = [
...
   {
     name: "push",
     description:
       "Provisions cloud resources with the latest local developments",
     subcommands: [
       {
-        name: "--y",
+        name: "-y",
         description: "Respond 'y' to all options",
       },
```
Comment (fedeci) [inline src/amplify.ts:68]: This should be an option actually and not a subcommand. Can you change that?

File: src/amplify.ts [hunk 2]
```diff
@@ -91,7 +96,7 @@ const completionSpec: Fig.Spec = {
         "Provisions cloud resources with the latest local developments",
       subcommands: [
         {
-          name: "--y",
+          name: "-y",
```
Comment (fedeci) [inline src/amplify.ts:99]: Same here

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/amplify.ts (hunk 1 above)
Comment (adilusman51) [reply src/amplify.ts]: Kindly review, I have made the requested changes

**Review by fedeci — APPROVED** [review]
Thanks!

**Comment by fedeci** [comment]
It seems that lint is failing, can you run `npm run lint:fix` please?

**Comment by adilusman51** [comment]
@fedeci Lint issues resolved

**Comment by fedeci** [comment]
Great, thanks!

**Pull request merged**